wiki/osu!wiki_Contribution_Guide/GitHub_Desktop/id.md

```diff
@@ -0,0 +1,105 @@
+# GitHub Desktop
+
+*Halaman utama: [Panduan Kontribusi osu!wiki](/wiki/owcg).*
+
+*Artikel ini merupakan lanjutan dari halaman utama* dan berasumsi bahwa anda akan menggunakan [Github Desktop](https://desktop.github.com). **Anda dapat menggunakan klien git lain dengan resiko anda sendiri.** Bahkan jika anda bekerja secara lokal, anda masih harus mengakses GitHub untuk membuat pull request supaya anda dapat menerapkan perubahan yang anda lakukan.
+
+## Memasang Github Desktop
+
+1\. Pasang [GitHub Desktop](https://desktop.github.com) jika anda belum melakukannya.
+
+2\. Ketika penginstal selseai, Github Desktop akan meminta anda untuk masuk, klik `Sign into GitHub.com`.
+
+3\. Isi petunjuknya saat muncul.
+
+4\. Lanjutkan ke [Cloning](#cloning).
+
+## Cloning
+
+1\. Klik `Clone a repository`.
+
+2\. Dengan ini daftar repositori anda akan ditampilkan, pilih repo `osu-wiki`.
+
+3\. Pilih lokasi clone. Tidak masalah dimana anda menaruhnya, tetapi selalu ingat di mana anda mengkloningnya.
+
+![](img/github-desktop-select-repo.jpg)
+
+4\. Klik `Clone`. Hal ini mungkin akan memerlukan waktu beberapa saat tergantung pada jaringan anda.
+
+5\. Lanjutkan ke [Branching](#branching).
+
+## Branching
+
+*Catatan: Hal ini sebenarnya opsional, tetapi sangati dianjurkan karena beberapa hal yang dapat anda cari sendiri*
+
+1\. Klik pada menu dropdown `Current branch`. Menu ini terletak di bagian atas menu.
+
+2\. Masukkan nama untuk branch ini. Disarankan untuk menggunakan nama artikel yang anda sunting sebagai nama branch diikuti dengan angka. Misalnya, menyunting artikel `Forums` anda akan memberikan nama branch-nya `forums1`.
+
+![](img/github-desktop-branch.jpg)
+
+3\. Lanjutkan ke [Penyuntingan](#editing).
+
+## Penyuntingan
+
+1\. Di dalam folder tempat anda mengkloning repo, arahkan ke file yang akan anda sunting. Artikel diatur oleh nama-nama folder sebagai nama artikel dan file `.md` sebagai locale-nya.
+
+2\. Setelah anda menemukan file yang benar, buka menggunakan teks editor favorit anda.
+
+3\. Anda sekarang dapat membuat perubahan anda. Saat membuat perubahan, coba ikuti [Kriteria Penyusunan Artikel](/wiki/ASC) sebisa mungkin.
+
+4\. Lanjutkan ke [Committing and pushing](#committing-and-pushing).
+
+### Memindahkan file yang ada
+
+1\. Gunakan Windows Explorer untuk memindahkan file.
+
+2\. Lanjutkan ke [Committing and pushing](#committing-and-pushing).
```

Comment:
you should probably translate "committing and pushing", as you've translated every title so far, but add the english term in brackets after the translated word

this applies to all "committing and pushing" in the article